postgres-persistence/src/main/java/com/netflix/conductor/postgres/PostgresDataSourceProvider.java

```diff
@@ -13,9 +13,11 @@
 package com.netflix.conductor.postgres;
 
 import com.google.common.util.concurrent.ThreadFactoryBuilder;
+import com.google.inject.internal.cglib.proxy.$Factory;
```

Comment:
Is this import needed?

Reply:
> Is this import needed?

Thanks. Removed. Don't even know how it got there.